docs/quick-start-guide.md

```diff
@@ -369,7 +362,7 @@ $SPARK_HOME/carbonlib/$CARBON_ASSEMBLY_JAR <carbon_store_path>
 ```
 ./bin/spark-submit \
 --class org.apache.carbondata.spark.thriftserver.CarbonThriftServer \
-$SPARK_HOME/carbonlib/carbondata_2.xx-x.x.x-SNAPSHOT-shade-hadoop2.7.2.jar \
+$SPARK_HOME/carbonlib/apache-carbondata-xxx.jar \
 hdfs://<host_name>:port/user/hive/warehouse/carbon.store
```

Comment:
This command will not work as thrift server doesn't take store location as argument , please check the class and update the same. suggest to test also.

Reply:
done